docs/cmd/tkn_pipeline_start.md

```diff
@@ -1,38 +1,55 @@
 ## tkn pipeline start
 
-Start the `pipeline`
+Start pipelines
 
 ### Synopsis
 
-Start the `pipeline` by creating a pipelinerun
+To start a pipeline, you will need to pass the following:
+
+- Resources
+- Parameters, at least those that have no default value
 
 ```
-tkn pipeline start NAME [flags]
+tkn pipeline start
 ```
 
-### Aliases
+### Examples
 
 ```
-start, trigger
+To run a Pipeline that has one git resource and no parameter.
+
+```shell
+$ tkn pipeline start --resource source=samples-git
+```
+
+To run a Pipeline that has one git resource, one image resource and
+two parameters (foo and bar)
+
+```shell
+$ tkn pipeline start --resource source=samples-git \
+	--resource image=my-image \
+	--param foo=yay \
+	--param bar=10
+```
 ```
 
 ### Options
```

Comment:
Just wondering why these options are getting print in the right format.
![image](https://user-images.githubusercontent.com/28615674/60681442-df558c80-9eac-11e9-9a52-b4d992f1a1cc.png)

Reply:
Ah interesting, there is a bug :joy: I'll look into why it does this…